docs/pbc/all/merchant-management/202407.0/marketplace/install-and-upgrade/tutorials-and-howtos/create-merchant-commission-collector-rule.md

```diff
@@ -0,0 +1,367 @@
+---
+title: "Create Merchant Commission Collector Rule"
```

Comment:
are the quotes really needed?

Reply:
Idk, some docs have it, and some don't :)